docs/installation.md

```diff
@@ -40,6 +48,26 @@ Note that by default, Jupyter Notebook will not be able to open a browser window
 WSL2 [does support](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gpu-compute) GPU compute, at least with nvidia cards, but setting it up [takes some work](https://developer.nvidia.com/cuda/wsl).
 :::
 
+
+(installation/m1)=
+
+#### macOS with M1 chips
+
+Apple Silicon (M1 mac, `osx-arm64`) is supported, but only through Rosetta because (as of August 2022) some upstream dependencies are not yet built on it.
+To configure `conda` to [use Rosetta](https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon) when creating an environment (or modifying one, such as installing new packages into it), prepend the command with `CONDA_SUBDIR=osx-64`:
+
+```shell-session
+$ CONDA_SUBDIR=osx-64 conda install -c conda-forge openff-toolkit
+```
+
+To make this setting the global default, update the conda config via
+
+```
+$ conda config --env --set subdir osx-64
+```
+
+Note that this will affect how `conda` behaves with other environments.
```

Comment:
```suggestion
Note that this will affect how Conda behaves with other environments.

[Rosetta]: https://support.apple.com/en-au/HT211861
[use Rosetta]: https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon
```

For consistency with the rest of the page...